src/frontend/src/pages/myfeeds.js

```diff
@@ -1,7 +1,26 @@
-import React, { useState } from 'react';
-import { Container, Box, Typography, TextField, Grid, Card, IconButton } from '@material-ui/core';
-import { AccountCircle, RssFeed, HelpOutline, Add } from '@material-ui/icons';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Card,
+  Container,
+  Dialog,
+  DialogContent,
+  DialogActions,
+  DialogContentText,
+  DialogTitle,
+  Grid,
+  IconButton,
+  Popover,
+  Typography,
+} from '@material-ui/core';
+import { AccountCircle, AddCircle, Delete, HelpOutline, RssFeed } from '@material-ui/icons';
 import { makeStyles } from '@material-ui/core/styles';
+import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+import PopupState, { bindTrigger, bindPopover } from 'material-ui-popup-state';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why is this necessary?

Reply:
Since `valid-url` is already included in our root-level `package.json`, I did not think it would be a good idea to also include it within our frontend-level `package.json`. As such, I added this line to prevent ESLint complaining about its lack of inclusion there.

https://github.com/Seneca-CDOT/telescope/blob/9cba5ed46c166f620c21f19f16baeccc383a44f6/package.json#L94

Reply:
I wonder if we should just disable this rule in our eslintrc, since we have a few places where it would be good to rely on deps futher up.

Reply:
Probably a good idea; should I file an issue?

Reply:
I'd do it in this, so you can remove that eslint ignore.

Reply:
Done: https://github.com/Seneca-CDOT/telescope/pull/933/commits/2ae7efe11944d53d23e43c897c561cf9948cfedb#diff-e4403a877d80de653400d88d85e4801aR108

Regarding ESLint changes: I also took the liberty of installing the `eslint-plugin-react-hooks` plugin, which I have been finding useful while learning about and working with React Hooks.

Reply:
> Can we break myfeeds.js up into different files? We have a bunch of separate components all being defined in this file, and it's making it hard to read the code.

Good idea, and done in https://github.com/Seneca-CDOT/telescope/pull/933/commits/2ae7efe11944d53d23e43c897c561cf9948cfedb:
- [DeleteFeedDialogButton](https://github.com/Seneca-CDOT/telescope/pull/933/commits/2ae7efe11944d53d23e43c897c561cf9948cfedb#diff-c1747a45c828916ed2e3274bdd5df4dc)
- [HelpPopoverButton](https://github.com/Seneca-CDOT/telescope/pull/933/commits/2ae7efe11944d53d23e43c897c561cf9948cfedb#diff-2b3be8acde3c2c96452d43f50b33ef2a)
- [ExistingFeedList](https://github.com/Seneca-CDOT/telescope/pull/933/commits/2ae7efe11944d53d23e43c897c561cf9948cfedb#diff-2117800b114a1b4cfaa8fb1e262360ea)